VideoLocker/res/layout/row_myvideo_course_list.xml

```diff
@@ -19,45 +19,36 @@
         android:layout_height="@dimen/course_detail_card_height"
         android:layout_gravity="bottom"
         android:background="@color/transparent_white_85"
-        android:orientation="vertical">
+        android:orientation="vertical"
+        android:paddingEnd="10dp"
+        android:paddingLeft="10dp"
+        android:paddingRight="10dp"
+        android:paddingStart="10dp"
+        android:paddingTop="10dp"
+        tools:targetApi="17">
 
         <org.edx.mobile.view.custom.ETextView
             android:id="@+id/course_name"
-            style="@style/bold_grey_text"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginBottom="8dp"
```

Comment:
Should we really remove this bottom margin as well? Perhaps it should also be moved to the parent's padding.